terraform/shared/shared.tf.json.template.py

```diff
@@ -174,6 +174,36 @@
                 }
             }
         },
+        **(
+            {
+                'aws_cloudformation_stack': {
+                    'chatbot': {
+                        'name': config.qualified_resource_name('chatbot'),
+                        'template_body': json.dumps({
+                            'AWSTemplateFormatVersion': '2010-09-09',
+                            'Description': 'AWS Chatbot Configuration',
+                            'Resources': {
+                                'SlackChannelConfiguration': {
+                                    'Type': 'AWS::Chatbot::SlackChannelConfiguration',
+                                    'Properties': {
+                                        'ConfigurationName': config.qualified_resource_name('chatbot'),
+                                        'GuardrailPolicies': 'arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess',
+                                        'IamRoleArn': '${aws_iam_role.chatbot.arn}',
+                                        'LoggingLevel': 'INFO',
+                                        'SlackChannelId': config.slack_integration['channel_id'],
+                                        'SlackWorkspaceId': config.slack_integration['workspace_id'],
+                                        'SnsTopicArns': '${aws_sns_topic.monitoring.arn}',
```

Comment:
The `monitoring` `aws_sns_topic` isn't declared in this file/directory, so I believe this needs to be referenced via a `data` resource.